File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SurveyModel.ts
import { SurveyDetailListModel } from "../ListModels/SurveyDetailListModel";


export class SurveyModel {
    surveyId: number;
    surveyCode: string;
    createDate: Date | null;
    serveyGroupId: number | null;
    surveyNameEn: string;
    surveyNameAr: string;
    isActive: boolean | null;
    details: SurveyDetailListModel[];
}

File: SofiForce.Models/Models/EntityModels/SurveyModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class SurveyModel
    {
        public string Id { get; set; }
        public int? SurveyId { get; set; }
        public string SurveyCode { get; set; }
        public DateTime? CreateDate { get; set; }
        public int? ServeyGroupId { get; set; }
        public string SurveyNameEn { get; set; }
        public string SurveyNameAr { get; set; }
        public bool? IsActive { get; set; }

        public List<SurveyDetailListModel> Details { get; set; } = new List<SurveyDetailListModel>();
    }
}
